docs/reference/troubleshooting/no-class-def-found-error.md

```diff
@@ -0,0 +1,9 @@
+---
+navigation_title: "NoClassDefFoundError"
+mapped_pages:
+  - https://www.elastic.co/guide/en/elasticsearch/client/java-api-client/current/no-class-def-found-error.html
+---
+
+# `NoClassDefFoundError`
+
+Check the [Known Issues](/release-notes/known-issues.md#known-issues-9-0-0)
```

Comment:
NoClassDefFoundError is very generic, we should be more precise so that users can know here if this applies, like "NoClassDefFoundError(org/apache/commons/logging/LogFactory)"